File: EducationApp.DataAccessLayer/Models/Filters/Order/OrderFilterModel.cs
using EducationApp.DataAccessLayer.Helpers.Base;
using EducationApp.DataAccessLayer.Models.Base;
using System.Collections.Generic;
using static EducationApp.DataAccessLayer.Entities.Enums.Enums;

namespace EducationApp.DataAccessLayer.Helpers.OrderFilterModel
{
    public class OrderFilterModel : BaseFilterModel
    {
        public SortOrder SortOrder { get; set; }
        public List< OrderStatusType> StatusOrder { get; set; }
        public long Id { get; set; }
    }
}

File: ClientApp/src/app/shared/models/order/OrderFilterModel.ts
import { OrderStatusType } from 'src/app/shared/enums/OrderStatusType';
import { SortOrderType } from 'src/app/shared/enums/SortOrderType';
import { BaseFilterModel } from 'src/app/shared/models/Base/BaseFilterModel';

export class OrderFilterModel extends BaseFilterModel {
    id:number;
    statusOrder: Array<OrderStatusType>;
    sortOrder: SortOrderType;

}